contrib-core/src/main/java/net/mostlyoriginal/api/utils/QuadTree.java

```diff
@@ -53,9 +55,20 @@ public QuadTree() {
      * Specify max tree bounds
      */
     public QuadTree(float x, float y, float width, float height) {
+        this(x, y, width, height, 16, 8);
+    }
+
+    /**
+     * Public constructor for initial {@link QuadTree}
+     * <p>
+     * Specify max tree bounds, bucket capacity and max depth
+     */
+    public QuadTree(float x, float y, float width, float height, int capacity, int depth) {
         bounds = new Container();
-        containers = new Bag<>(MAX_IN_BUCKET);
+        containers = new Bag<>(capacity);
```

Comment:
it would be more informative to pass `maxInBucket` instead of `capacity`

Reply:
Agreed & changed